samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/author/AuthorBean.java

```diff
@@ -963,11 +964,15 @@ public void setGroupFilterEnabled(boolean groupFilterEnabled) {
   }
 
 	public Boolean questionHasRubric(Long assessmentId, Long questionId, boolean isPublished) {
-		if(!isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO, assessmentId + "." + questionId)){
-			return Boolean.TRUE;
-		}
-		if(isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO,RubricsConstants.RBCS_PUBLISHED_ASSESSMENT_ENTITY_PREFIX + assessmentId + "." + questionId)){
-			return Boolean.TRUE;
+		try {
+			if(!isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO, assessmentId + "." + questionId)){
+				return Boolean.TRUE;
+			}
+			if(isPublished && rubricsService.hasAssociatedRubric(RubricsConstants.RBCS_TOOL_SAMIGO, RubricsConstants.RBCS_PUBLISHED_ASSESSMENT_ENTITY_PREFIX + assessmentId + "." + questionId)){
+				return Boolean.TRUE;
+			}
+		} catch(HttpClientErrorException hcee) {
+			log.debug("Current user doesn't have permission to get a rubric");
```

Comment:
log.debug should print the exception message i.e. 
`log.debug("Current user doesn't have permission to get a rubric, {}" hcee.getMEssage());`

Reply:
I'll add it, but I believe it was just 'null'